Sources/MapboxMaps/Foundation/Camera/CameraAnimationProtocols.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
```

Comment:
Let's split this file up?

Reply:
One file for the `CameraAnimatorDelegate` and the other for the `AnimationOwner` + `AnimationOwnerProtocol`